.github/workflows/update-changeset-prs.yaml

```diff
@@ -31,17 +54,7 @@ jobs:
         # We want to commit the yarn.lock changes
         run: yarn install --no-immutable
 
-      - name: Get last commit message
-        id: last-commit
-        run: |
-          echo "message=$(git log -1 --pretty=%s)" >> $GITHUB_OUTPUT
-          echo "author=$(git log -1 --pretty=\"%an <%ae>\")" >> $GITHUB_OUTPUT
-
       - name: Amend previous commit
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
-          commit_author: ${{ steps.last-commit.outputs.author }}
-          commit_message: ${{ steps.last-commit.outputs.message }}
-          commit_options: '--amend --no-edit'
-          push_options: '--force'
-          skip_fetch: true
+          commit_message: 'chore: update yarn.lock'
```

Comment:
should we also set `file_pattern` to restrict this to only the yarn.lock file(s) ?

Reply:
do you need the whole "is the previous commit message a `chore: update yarn.lock` ? because if the yarn lock regen shows no changes, there's nothing to commit.